checkClock/db/migrate/20220113230211_create_admins.rb

```diff
@@ -1,10 +0,0 @@
-class CreateAdmins < ActiveRecord::Migration[6.1]
```

Comment:
It's not necessary to delete this migration, that's not the common approach in production applications

For now, it's okay,  but wanted to make the call out here to let you know that usually, migrations, once they are running in our app, are not deleted nor updated, instead a new migration is created to make the proper changes to our db tables 😄 👍🏽